README.rst

```diff
@@ -36,6 +36,8 @@ Include the Flow Control component in the content unit you want to control acces
 * Enter the problem locator ids (as many as required) to evaluate the condition.
 * Select an action to apply when the condition is met.
 
+⚠️ Since Open edX Olive some the features could fail because to changes in MFE integration UI.
```

Comment:
```suggestion
⚠️ Since the Open edX Olive release, some features could fail because of the integration of MFE.
```

And I suggest adding this message before the installation instructions. To make it quickly visible.

Reply:
Also, if the only MFE that causes problems is learning MFE, it is better if we are specific in that.